spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+RSpec.describe 'User index integration spec', type: :system do
+  describe 'index page' do
+    let!(:user_two) { User.create name: 'Mufti menk', posts_counter: 0 }
+    let!(:user_three) { User.create name: 'Dr Zakir', posts_counter: 0 }
+    it 'should render the user name' do
+      visit users_path
+      expect(page).to have_content(user_two.name)
+    end
+    it 'should render the users photo' do
+      visit users_path
+      expect(page).to have_content(user_two.photo)
+    end
+    it 'should render the number of posts' do
+      visit users_path
+      expect(page).to have_content("Number of posts #{user_two.posts.size}")
+    end
```

Comment:
- [ ] In the testing requirements for the User index page, it is expected that you write tests that assert an expected outcome for all users - not a single user as you have done here. Please see the requirements below and adjust each of your test to cater to all the users 🙏 
![image](https://github.com/muhammedb8826/blog-app/assets/46839548/38628f38-1dfc-4180-af1b-9664be7ac731)